scripts/clustering.py

```diff
@@ -73,11 +93,9 @@ def main(fmat, kmat, ftags, prefix, fcolor, dimension, pc1, pc2, algorithm, adte
     # the characteristic bandwidth of the data
     sigma_kij = density_model.bandwidth
     rho = density_model.evaluate_density(proj)
-    meanrho = np.mean(rho)
 
-    algorithm = str(algorithm)
     # now we do the clustering
-    if algorithm == 'dbscan' or algorithm == 'DBSCAN':
```

Comment:
Since 'dbscan' is mentioned as the option in the docstring I've removed the option to specify 'DBSCAN'